### PR TITLE
Make <details> open if there is an #id in the URL

### PR DESCRIPTION
**Pull request by henryhadlow**

### CODE REVIEW

File: app/views/pages/_home_responded_petitions.html.erb [hunk 1]
```diff
@@ -7,7 +7,7 @@
         <h3><%= link_to petition.action, petition_path(petition), class: "threshold-petition-title" %></h3>
         <p class="intro">The government responded</p>
         <blockquote class="pull-quote"><%= simple_format(petition.government_response.summary) %></blockquote>
-        <p><%= link_to "Read the response in full", petition_path(petition) %></p>
+        <p><%= link_to "Read the response in full", petition_path(petition, show_response: "✓", anchor: 'response-threshold') %></p>
```
Comment (oskarpearson) [inline app/views/pages/_home_responded_petitions.html.erb]: Does this work on browsers that don't support unicode? I'm not sure of our browser coverage expectations.

**Comment by oskarpearson** [comment]
I'm probably not the best person to review the browser polyfill - but other than my query above, I think this is fine :)

File: app/views/pages/_home_responded_petitions.html.erb (hunk 1 above)
Comment (henryhadlow) [inline app/views/pages/_home_responded_petitions.html.erb]: Not sure. I'll remove the unicode character.

Comment (pixeltrix) [inline app/views/pages/_home_responded_petitions.html.erb]: @oskarpearson are there browsers that we support that don't support unicode? Even IE6 has UTF-8 support.

**Pull request merged**